### DISCUSSION TITLE
Trouble with login to the admin tool

### DISCUSSION THREAD

**Discussion by juhovainio** [rcl-ssl/docs#29] [Q&A]
Hi! I'm trying to access our RCL admin page in order to renew some of the certificates but now every login attempt seems to be failing for the admin level users, with the error message stating that the user accounts do not exists in tenant rclapp.com. Any idea how I should go about solving this issue?

Here is the error message from the auth flow:

Request Id: 047d3311-9d42-4bb1-8671-251e83ea4a01
Correlation Id: 5c8cea5d-5be3-4870-ae15-540d047d5244
Timestamp: 2022-04-27T08:31:10Z
Message: AADSTS50020: User account [redacted] from identity provider 'https://sts.windows.net/5cad934d-7b0e-4789-8584-ee002452f2cc/' does not exist in tenant 'rclapp.com' and cannot access the application 'e95a9f9b-3abe-41bc-816f-79f124a8b3da'(Ray Consulting Limited) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure Active Directory user account.

**Comment by rcladmin** [comment]
Is the issue fixed now?

**Reply by juhovainio** [reply]
Great, thank you, login seems to be working now. Any idea if there was something that had happened on our side causing this?

Also, now when trying to manually update the previously added App Service certificates produces the following error which was not existing before. Could this also be related to the issue?

```
Error:Could not manually update certificate : admin-demo.mymobiletutor.eu .
BadRequest : Could not bind the PFX certificate : admin-demo.mymobiletutor.eu to the app service : cyf-admin/staging.
Failure in AzureService in AzureHostnameBindingService : Get.
Failed response : NotFound in AzureHostnameBindingService : Get. . SSL Azure App Service Bind Certificate - RCL V1 API
Information:Waiting for process to complete ****
Connection Id : 738
```

**Reply by rcladmin** [reply]
looking into this

**Reply by rcladmin** [reply]
You can delete the certificate and recreate it. Let me know if it works. The update function should work when the certificates nears expiration.
